webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/InventoryApi.scala

```diff
@@ -90,20 +85,12 @@ class InventoryApi (
     val restExtractor = restExtractorService
     val actionName = "queueInformation"
     def process0(version: ApiVersion, path: ApiPath, req: Req, params: DefaultParams, authzToken: AuthzToken): LiftResponse = {
-      val saturated = inventoryProcessor.isQueueFull.catchAll(err =>
-        InventoryProcessingLogger.error(err.fullMsg) *> true.succeed
-      ).runNow
       val json = (
-                    ("queueMaxSize"   -> inventoryProcessor.maxQueueSize)
-                  ~ ("queueSaturated" -> saturated)
+                    ("queueMaxSize"   -> Int.MaxValue)
+                  ~ ("queueSaturated" -> false)
```

Comment:
I kept that to not break the API, but it makes little sense. 
I propose to:
- do that for up to version 13
- create a version 14 with info `{"incomingInventories": XXX}` the number of inventories in incoming+accepted-nodes-updates

It can be done after merging this PR.